spec/models/group_spec.rb

```diff
@@ -81,15 +81,7 @@
 
     it 'should roles removed when removing group membership' do
       expect(user).to be_member_of project
-      member.destroy
```

Comment:
Destroying a member of a group is not the same as destroying the group. There were two different test cases before this commit. Now it is only one. Testing the deletion of a group member is gone and I believe it should be there.

Reply:
Well spotted, I was sure they were duplicate when I removed them